### ISSUE TITLE
SA1110 (OpeningParenthesisMustBeOnDeclarationLine) reporting false positives

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#545]
SA1110 is getting reported for the `(` token following `TriviaList` in the following code:

```csharp
private static SyntaxTriviaList XmlLineStart()
{
    return SyntaxFactory.TriviaList(
        SyntaxFactory.ElasticMarker,
        SyntaxFactory.DocumentationCommentExterior("/// "));
}
```

**Comment by macpak** [comment]
Ok, I see the problem.
